charts/values.yaml

```diff
@@ -103,14 +117,24 @@ serviceAccount:
 # Configure options for network policy
 # ref: https://kubernetes.io/docs/concepts/services-networking/network-policies
 networkPolicy:
-  enabled: false
+  enabled: true
```

Comment:
Should we by default disable the network policy?

Reply:
Yes we should I think as it can lead to some issue if not set properly.
This is a standard behaviour in many helm charts